tools/linktool

```diff
@@ -361,6 +388,86 @@ def show_previous(link_uuid):
     previous_uuids = get_previous_links(link_uuid)
     print "Previous link UUIDs: {0}".format(', '.join(previous_uuids))
 
+def remove_from_chains(link_uuid):
```

Comment:
Can you add the SQL to delete the now unused MicroServiceChainLink, MicroServiceChainLinkExitCode, and possibly TasksConfig and StandardTasksConfig from this?  The second two are probably hard to detect, so printing a "TODO check if these need to be removed" would be fine.

`replace_link_with_new_link` and `replace_link_with_existing_task` also need this.